building/glass/aptbranch.py

```diff
@@ -73,3 +43,34 @@ def export_key(keyid, armor=False):
     if not result.strip():
         raise Exception("empty result from gpg for keyid: '%s'" % keyid)
     return result
+
+
+class AptBranchConfig:
```

Comment:
Maybe just call this `Config`? It's already implied to be an _apt branch_ config by way of the module that it's in.